.docker/nginx/conf.d/nginx.conf

```diff
@@ -1,17 +1,30 @@
-# from https://symfony.com/doc/3.4/setup/web_server_configuration.html#nginx
+# from https://symfony.com/doc/4.4/setup/web_server_configuration.html
 server {
-    server_name ${NGINX_HOST};
-    root /var/www/web;
+    server_name ${NGINX_HOST} admin.${NGINX_HOST};
```

Comment:
@mathieu-ducrot variable in ngnix config so we can have only one file

Reply:
@nicolas-bastien when log in nginx container and echoing $NGINX_HOST i get nothing so i'm not sure if this could work on production. It's already weird that it work localy.